src/components/Community/Contribute/index.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { logEvent } from '../../../utils/ga'
+
+import CommunityButton from '../Button'
+import CommunityBlock from '../Block'
+import CommunitySection from '../Section'
+
+import data from '../data'
+
+import { Item, Items, Wrapper } from '../styles'
+
+const { description, mobileDescription, title } = data.section.contribute
+
+const logPR = () => logEvent('community', 'contribute-pr')
+const logBlogpost = () => logEvent('community', 'contribute-blogpost')
+const logTalk = () => logEvent('community', 'contribute-talk')
+const logAmbassador = () => logEvent('community', 'contribute-ambassador')
+
+export default function CommunityContribute({ theme }) {
+  return (
+    <Wrapper>
+      <CommunitySection
+        anchor="contribute"
+        background="/static/img/community/contribute_bg.jpg"
+        color={theme.color}
+        description={description}
+        icon="/static/img/community/contribute.svg"
+        mobileDescription={mobileDescription}
+        title={title}
+      >
+        <Items>
+          <Item>
+            <CommunityBlock
+              title="Make a PR"
+              action={
+                <CommunityButton
+                  theme={theme}
+                  href="https://github.com/iterative/dvc"
+                  target="_blank"
+                  rel="noreferrer noopener"
+                  onClick={logPR}
+                >
+                  Go to Github
+                </CommunityButton>
+              }
+            >
+              Become DVC contributor and let us build something great together.
+            </CommunityBlock>
+          </Item>
+          <Item>
+            <CommunityBlock
+              title="Write a blogpost"
+              action={
+                <CommunityButton
+                  theme={theme}
+                  href="mailto:info@dvc.org?subject=I want to write a blogpost!"
+                  target="_blank"
```

Comment:
nitpick: the only way of making mailto links open in a new tab is using `window.open`.

Reply:
It's ok not to open new page if the user use external email app. We need it only for browser email client like gmail.

@shcheklein can you check if the links are now opened in the new tab in your case please?